kubetest/anywhere.go

```diff
@@ -27,13 +27,16 @@ import (
 	"time"
 )
 
+var defaultKubeadmCNI = "weave"
```

Comment:
I originally wanted to, but the code needs to [take the address](https://github.com/kubernetes/test-infra/pull/2771/files#diff-c2755283c254aab8daa60c298e3e79e5R100) of the string when setting the default, so we can only use a var.